atlp-rwanda/falcons-ec-fe
# # 185106775 Seller Two Factor Auth

**What does this PR do?**

- This PR implements the two factor authentication feature

**Description of Task to be completed?**

- A seller should be able to authenticate with otp code

**How should this be manually tested?**

- Clone this project run the script "npm run dev" then go to ''signin" then signin as a seller ==> "users/verify"

**Any background context you want to provide?**

- N/A

**What are the relevant pivotal tracker/Trello stories?**

- Tracker: 185106775

**Screenshot**

<img width="1440" alt="Screenshot 2023-06-01 at 05 45 17" src="https://github.com/atlp-rwanda/falcons-ec-fe/assets/90443432/c9b36628-d5b9-4c53-a171-e0ad0b5e3757">

<img width="1440" alt="Screenshot 2023-06-01 at 05 45 27" src="https://github.com/atlp-rwanda/falcons-ec-fe/assets/90443432/0eec91b6-6e5d-4399-9149-47e201a219a5">


https://github.com/atlp-rwanda/falcons-ec-fe/assets/90443432/34be9bf8-294f-4bb7-aaea-63bd3084b9a2

**Questions**

- None
